src/index.js

```diff
@@ -38,3 +37,19 @@ export const renderReactWithAphrodite = (name, component) => hypernova({
     return component;
   },
 });
+
+export const renderReactWithAphroditeStatic = (name, component) => hypernova({
+  server() {
+    return (props) => {
+      const { html, css } = StyleSheetServer.renderStatic(() => (
+        ReactDOMServer.renderToStaticMarkup(React.createElement(component, props))
+      ));
+
+      const style = `<style data-aphrodite>${css.content}</style>`;
+
+      return `${style}\n${html}`;
```

Comment:
Should be fine. `serialize` is for sending down the JSON to the client which we don't do when static rendering.